src/pylptlib/lpt.py

```diff
@@ -23,7 +23,19 @@
 
 from .error_codes import ERROR_CODES
 
-_dll_path = r"C:\s4200\sys\bin\lptlib.dll"
+# Currently, the lptlib.dll can only be loaded if it is installed in the standard Clarius installation path.

```

Comment:
Doesn't this contradict the code below? I.e. it could also be in the current working directory (which does not necessarily need to be the directory of the exe)

Reply:
Changed the comments